app/controllers/coronavirus_form/offer_care_qualifications_controller.rb

```diff
@@ -31,8 +29,14 @@ def submit
       log_validation_error(invalid_fields)
       render controller_path
     elsif session["check_answers_seen"]
+      session[:offer_care_type] = @form_responses[:offer_care_type]
+      session[:offer_care_qualifications] = @form_responses[:offer_care_qualifications]
+      session[:offer_care_qualifications_type] = @form_responses[:offer_care_qualifications_type]
```

Comment:
Perhaps this could be extracted into a method?

Reply:
Perhaps every controller could be forced to implement a `update_session` (name tbd) method? That way the logic only needs to be added once, and it there would less chance of making copy and paste errors?

Reply:
Good idea, I've put a `update_session_store` method into each controller that updates the session store.